Runtimes/Supplemental/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.29)
+
+project(SwiftRuntime LANGUAGES Swift C CXX)
+
+include(ExternalProject)
+
+set(SwiftRuntime_SWIFTC_SOURCE_DIR "${PROJECT_SOURCE_DIR}/../../")
+
+foreach(lib ${Swift_ENABLE_RUNTIMES})
+  string(TOLOWER ${lib} name)
+  set(SwiftRuntime_ENABLE_${name} YES)
+endforeach()
+
+if(SwiftCore_DIR)
+  set(SwiftCore_DIR_FLAG "-DSwiftCore_DIR=${SwiftCore_DIR}")
+endif()
+
+if(CMAKE_MAKE_PROGRAM)
+  set(MAKE_PROGRAM_FLAG "-DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}")
+endif()
+
+set(COMMON_OPTIONS
+  -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
+  -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
```

Comment:
I added 
```
  -DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}
  -DCMAKE_INSTALL_NAME_DIR=${CMAKE_INSTALL_NAME_DIR}
  -DCMAKE_BUILD_WITH_INSTALL_NAME_DIR=${CMAKE_BUILD_WITH_INSTALL_NAME_DIR}
```

In my synchronization PR so the supplemental super build could pass those through (assuming each uses the same install destination). Not sure if that's the best approach but it's similar to install_prefix